### ISSUE TITLE
[Bug]: Querynode crash when loading a large collection "invalid memory address or null pointer dereference".

### ISSUE THREAD

**Issue by simon-fauconnier** [milvus-io/milvus#16144]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: 2.0.1
- Deployment mode(standalone or cluster): cluster
- SDK version(e.g. pymilvus v2.0.0rc2): pymilvus v2.0.1
- OS(Ubuntu or CentOS): Ubuntu
- CPU/Memory: 32 cores / 256Gb
- GPU: /
- Others: milvus is deployed with helm chart version 3.0.10
```

### Current Behavior

# Issue

When I load a collection of more than 12.5M entities, the querynode crashes with a panic : 

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x1ae25ae]

goroutine 154 [running]:
github.com/milvus-io/milvus/internal/querynode.(*segmentLoader).loadSegment(0xc0006ac360, 0xc05df77a98, 0xc000000002, 0x2, 0x2)
	/go/src/github.com/milvus-io/milvus/internal/querynode/segment_loader.go:151 +0x15ae
github.com/milvus-io/milvus/internal/querynode.(*watchDmChannelsTask).Execute(0xc00034e060, 0x34c06d0, 0xc000327c80, 0x0, 0x0)
	/go/src/github.com/milvus-io/milvus/internal/querynode/task.go:290 +0xf3f
github.com/milvus-io/milvus/internal/querynode.(*taskScheduler).processTask(0xc000327cc0, 0x34d35d8, 0xc00034e060, 0x34d1750, 0xc00014e5f0)
	/go/src/github.com/milvus-io/milvus/internal/querynode/task_scheduler.go:61 +0x173
github.com/milvus-io/milvus/internal/querynode.(*taskScheduler).taskLoop(0xc000327cc0)
	/go/src/github.com/milvus-io/milvus/internal/querynode/task_scheduler.go:78 +0x15a
created by github.com/milvus-io/milvus/internal/querynode.(*taskScheduler).Start
	/go/src/github.com/milvus-io/milvus/internal/querynode/task_scheduler.go:86 +0x5c
```

Full querynode log : [milvus-querynode-6d8fffd5c9-vjs6m.log](https://github.com/milvus-io/milvus/files/8324928/milvus-querynode-6d8fffd5c9-vjs6m.log)

Helm chart values : [milvus.yml](https://github.com/milvus-io/milvus/files/8324884/milvus-yml.txt)

## Code to reproduce the error:

```
from pymilvus import (
    connections,
    FieldSchema,
    CollectionSchema,
    DataType,
    Collection,
)
from random import randint, random

MILVUS_PORT = 19530

milvus = connections.connect("default", host="localhost", port=MILVUS_PORT) #Port forward

schema = CollectionSchema(
  fields = [
      FieldSchema(
          name       = "id",
          dtype      = DataType.INT64,
          is_primary = True,
      ),
      FieldSchema(
          name  = "data",
          dtype = DataType.FLOAT_VECTOR,
          dim   = 128,
      ),
  ],
  description="Test collection"
)

collection = Collection(
    name   = "test",
    schema = schema
)

# create index if it does not exist yet
if len(collection.indexes) == 0:
  index_param = {
      "metric_type": "IP",
      "index_type": "HNSW",
      "params": {
          "M": 32,
          "efConstruction": 200,
      },
  }
  collection.create_index(
      field_name="data",
      index_params=index_param,
  )

for x in range(130):
  entities = [
      [int(randint(0, 9_223_372_036_854_775_807)) for _ in range(1000)],
      [[random() for _ in range(128)] for _ in range(100000)],
  ]
  insert_result = hello_milvus.insert(entities)

collection.load() #CRASH
```

### Expected Behavior

When I load the collection, the querynode should not panic (or maybe give a better error message)

### Steps To Reproduce

```markdown
1. Milvus is installed with helm chart (default settings) 
2. Create and fill a new collection
3. Load the collection ( collection.load() )
4. The querynode crashes instantly
```

### Anything else?

- There is no panic with the hello milvus example (with 20M entities)
- The same collection but with 12M entities is loaded without panic

**Comment by yanliang567** [comment]
sounds like a known issue in 2.0.1, which was fixed in master. Could you please retry with the master build or wait for 2.0.2 release?
/assign @simon-fauconnier 
/unassign

**Comment by zhuyaguang** [comment]
me too ,when restart milvus ,load collection crash 
<img width="996" alt="image" src="https://user-images.githubusercontent.com/8857976/159603107-a7f1427b-09be-40ca-921d-4ec2995c3913.png">

**Comment by simon-fauconnier** [comment]
> sounds like a known issue in 2.0.1, which was fixed in master. Could you please retry with the master build or wait for 2.0.2 release?

Thanks, do you have a release date for version 2.0.2 with the helm chart?

**Comment by simon-fauconnier** [comment]
Fixed in milvus 2.0.2 Thanks !
